src/Clerk.ts

```diff
@@ -194,6 +200,14 @@ export default class Clerk {
   }
 
   expressWithSession({ onError }: MiddlewareOptions = { onError: this.defaultOnError }): (req: Request, res: Response, next: NextFunction) => Promise<void> {
+    function isAuthV2Request(clerk: Clerk, token: undefined | string): boolean {
+      if (!token) {
+        return false
+      }
+
+      return clerk.decodeToken(token).hasOwnProperty('sid');
```

Comment:
:wrench: In Ruby we decide if it's an authv2 token based on if `iss.starts_with("https://clerk")`. I suggest to do the same here, as it's more explicit than simply checking if `sid` is present (which can be present from 3rd-party tokens as well).

Reply:
Correct